core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/InstanceStatus.java

```diff
@@ -10,6 +10,7 @@
     SERVICES_UNHEALTHY,
     WAITING_FOR_REPAIR,
     STOPPED,
+    STARTED,
```

Comment:
SERVICES_RUNNING should do the work I think. Unfortunately the name is misleading, but it means agents is up and running on the machine, I think after start we have the same state.

Reply:
STARTED and SERVICES_RUNNING are very different states. STARTED means the cloud-instance is STARTED, and has no information about the SERVICES.

Reply:
I can't see any flow in the new code which starts CM or Salt or other services, but the next step is to commission them in CM after we started nodes on provider. I also can't see any logic rely on this STARTED status. I think we can remove it.

Reply:
InstanceStatus combines cloud-provider and service health status.
STARTED indicates that the node is started on the cloud-provider side.

For a customer - seeing instances in STARTED state means wasted resources, and is the first stage of a stop-start upscale. That is where this state gets set. I don't think this can be removed.